tools/build.py

```diff
@@ -155,13 +154,7 @@ def generate_build_options(arguments):
     build_options.append('-DMEM_HEAP_SIZE_KB=%d' % arguments.mem_heap)
     build_options.append('-DPORT_DIR=%s' % arguments.port_dir)
 
-    if os.path.isabs(arguments.profile):
-        profile = arguments.profile
-    else:
-        profile = os.path.join(PROFILE_DIR, arguments.profile + '.profile')
-
-    build_options.append('-DFEATURE_PROFILE=%s' % profile)
-
+    build_options.append('-DFEATURE_PROFILE=%s' % arguments.profile)
```

Comment:
No, because this check is performed [here](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/CMakeLists.txt#L195).
```cmake
if (NOT IS_ABSOLUTE ${FEATURE_PROFILE})
  set(FEATURE_PROFILE "${CMAKE_CURRENT_SOURCE_DIR}/profiles/${FEATURE_PROFILE}.profile")
endif()
```